File: prescricao/src/main/java/br/com/basis/madre/prescricao/domain/Paciente.java
package br.com.basis.madre.prescricao.domain;

import java.io.Serializable;
import java.time.LocalDate;

import org.springframework.data.elasticsearch.annotations.Document;
import org.springframework.data.elasticsearch.annotations.Field;
import org.springframework.data.elasticsearch.annotations.FieldType;

import lombok.Data;

@Data
@Document(indexName = "madre-prescricao-paciente")
public class Paciente implements Serializable {
	
	private static final long serialVersionUID = 1L;

	private Long id;

	@Field(type = FieldType.Text)
	private String prontuario;
	
    @Field(type = FieldType.Text)
    
    private String nome;

    @Field(type = FieldType.Date)
    private LocalDate dataDeNascimento;
	

}

File: frontend/src/app/internacao/models/paciente.ts
export interface IPaciente {
    id?: number;
    nome?: string;
    nomeSocial?: string;
    dataDeNascimento?: Date;
    sexo?: string;
    nomeDaMae?: string;
    numeroCartaoSUS?: string;
}

export class Paciente implements IPaciente {
    public id?: number;
    public nome?: string;
    public nomeSocial?: string;
    public dataDeNascimento?: Date;
    public sexo?: string;
    public nomeDaMae?: string;
    public numeroCartaoSUS?: string;
}
